_data/changelogs/component-identifier.yml

```diff
@@ -2,6 +2,14 @@ title: Identifier
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Updated the screen reader readout to say "Official" instead of "An official".
+    summaryAdditional: Users should update their markup to improve the screen reader experience.
```

Comment:
## Optional change

Should we highlight why we made the change? Something like:

```suggestion
    summaryAdditional: This change prevents screen reader users from mishearing "Unofficial" instead of "An official". Users should update their markup to improve the screen reader experience.
```
Feel free to change the language.

Reply:
Good note. I updated it to be more similar to both the release notes and the related [banner changelog](https://designsystem.digital.gov/components/banner/#latest-updates) (issue #5925).